pom.xml

```diff
@@ -188,6 +188,11 @@
 				</exclusions>
 			</dependency>
 			<dependency>
+				<groupId>com.netflix.governator</groupId>
```

Comment:
https://github.com/Netflix/eureka/pull/606 will hopefully remove this dependency.